examples/2_Intermediate/stage_two_optimization_finitebuild.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env python
+r"""
+In this example we solve a FOCUS like Stage II coil optimisation problem: the
```

Comment:
In this docstring, how about saying something about finite build, discuss how the parameter space for this optimization includes the Fourier modes of alpha(theta), and compare and contrast this example with stage_two_optimization.py. It would also be nice to add a reference to the Singh paper here (I see there is a reference in the source, but one here too would be useful to help explain how this example works.)